### PR TITLE
HPCC-26378 Label Kibana for eclwatch link page

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: Rodrigo Pastrana <Rodrigo.Pastrana@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by rpastrana** [comment]
@GordonSmith please take a look, this is meant to spark further discussion.
This is the pre-existing elastic4hpcclogs chart.
We're now enabling kibana's UI link to be included in eclwatch, and overwriting the "landing page" via the path label.
Notice I did not provide connectivity info, since the back-end url info is dynamic and accessible from k8s api. I believe the front-end address has to be determined at deployment time as well. We can have a chat is easier to discuss.

File: helm/managed/logging/elastic/values.yaml
Comment (GordonSmith) [inline helm/managed/logging/elastic/values.yaml:104]: @rpastrana I think they should all be `hpcc.eclwatch.io` and no `hpcc.eclwatch.link.io`?
@wangkx I would suggest the following ESP Method:
```
hpccAnnotations("eclwatch.io") => [{"enabled": "true", "description": "Provides view in to HPCC component logs", label: "HPCC Logs via Kibana", "path": "/app/discover"}];
```

Comment (GordonSmith) [reply helm/managed/logging/elastic/values.yaml:104]: Maybe "hpccAnnotationNamespaces" might be better?

**Comment by rpastrana** [comment]
@wangkx / @GordonSmith  Please do formal review. We might not merge until ESP work is completed.

**Review by GordonSmith — APPROVED** [review]

**Comment by rpastrana** [comment]
@wangkx any comments before we merge?

**Comment by wangkx** [comment]
@rpastrana looks fine for me.

**Comment by rpastrana** [comment]
@ghalliday please merge

**Pull request merged**
